### PR TITLE
Manage 'huge data'

### PR DESCRIPTION
**Pull request by hillaryj**
Changes the parser to allow for parsing “huge” data files in excess of
9.5MB.

### CODE REVIEW

File: regml.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 import click
 from lxml import etree
+from lxml import objectify
```
Comment (willbarton) [inline regml.py]: If we're switching to lxml's `objectify` API, do we still need the `etree` import?

File: regml.py [hunk 2]
```diff
@@ -494,13 +505,21 @@ def apply_through(cfr_title, cfr_part, through=None):
...
             './{eregs}preamble/{eregs}effectiveDate').text
         applies_to = xml_tree.find(
             './{eregs}changeset').get('leftDocumentNumber')
+        if applies_to is None:
+            # Major problem here
+            print(colored("Error locating"),
+                  colored("leftDocumentNumber", attrs=['bold']),
+                  colored("attribute in"),
+                  colored("{}".format(doc_number), 'red',
+                          attrs=['bold']))
```
Comment (willbarton) [inline regml.py:521]: Should the function return after this error?

File: regml.py (hunk 1 above)
Comment (willbarton) [inline regml.py]: More I think about it, the more I think we should stick with etree for now... Don't want to introduce potential API weirdness (even though I don't think there be any).

You should be able to accomplish the same thing with:

```
parser = etree.XMLParser(huge_tree=True)
tree = etree.fromstring(xml_str, parser)
```

File: regml.py (hunk 2 above)
Comment (hillaryj) [inline regml.py:521]: Yes, it should - thanks!

**Comment by willbarton** [comment]
HUGE!

n9k354xcv1nquy6jpflg

**Pull request merged**